### DISCUSSION TITLE
Flask-SqlAlchemy-Plugin

### DISCUSSION THREAD

**Discussion by poojan-2601** [microsoft/vscode-pylint#242] [Q&A]
Hi, I am currently using pylint extension in vscode for my flask application, and it works perfectly, however when I want to use it for the model files in the project which uses Flask-SqlAlchemy. I have pip installed the pylint-flask-sqlachemy plugin in my environment, hence I check my files from the command line however I would also love to do the same from my editor.

**Comment by karthiknadig** [comment]
Have you tried adding this to settings? `"pylint.args" : ["--load-plugins", "pylint_flask_sqlalchemy"]`

**Reply by poojan-2601** [reply]
Thanks this was great!!!.
